server/api/field.go

```diff
@@ -35,9 +35,8 @@ type Field struct {
 	ModalLabel string `json:"modal_label"`
 
 	// Select props
-	SelectRefreshOnChangeTo []string       `json:"refresh_on_change_to,omitempty"`
-	SelectSourceURL         string         `json:"source_url,omitempty"`
```

Comment:
See my previous comment about lookups, 2/5 worth considering `SelectSource = Call{...`

Reply:
@levb Okay, the current structure stems from a screenshare conversation you and I had before the holidays. The reason we had it reuse the same call as the form, is because then the app's `function` can expect the appropriate form values defined in the respective form, for any/all of the form's lookup calls. `SelectSourceURL` has been removed in this PR, in favor of just using the form's `Call`.